ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -131,7 +131,7 @@ export const DataUserCodeOfConduct = fp.flow(withUserProfile(), withNavigation)(
       const duccVersion = getLiveDUCCVersion();
       profileApi().submitDataUseAgreement(duccVersion, initials).then((profile) => {
         this.props.profileState.updateCache(profile);
-        this.props.navigate(['/']);
+        history.back();
```

Comment:
navigates back to Registration Dashboard, Profile, or DAR

navigating back to AAR uses a separate function (see just above here)